qkit/measure/timedomain/sequence_library.py

```diff
@@ -0,0 +1,122 @@
+"""
+sequency_library.py
+A. Stehli (04/2018)
+
+This library provides the measurement sequences for standard qubit experiments.
+Currently this includes:
+    rabi experiment
+    T1 measurments
+    Ramsey experiments
+    Spin/Hahn echo
+"""
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+import pulse_sequence as ps
+
+
+def compensate():
```

Comment:
Empty function?

Reply:
Does not belong in sequence_library anymore.